ui/app/AppLayouts/Wallet/panels/SearchableAssetsPanel.qml

```diff
@@ -39,6 +39,14 @@ Control {
         searchBox.text = ""
     }
 
+    QtObject {
+        id: d
+        readonly property int delegateHeight: 60
```

Comment:
This is kind of duplicated magic number, here and in `TokenSelectorAssetDelegate`. Changing it in a delegate will break the list as it's fixed also here. Additionally the same thing is duplicated in `SearchableCollectiblesPanel`, increasing risk of accidental regression.

Moreover, currently the height is fixed, what is not inline with the design:

Current impl:
![image](https://github.com/user-attachments/assets/abbe70b8-0cd3-4a54-8ed3-08d0420ead5a)

Design:
![image](https://github.com/user-attachments/assets/591d907e-30f7-4e29-95fd-e3b39805bfe1)

Generally `SearchableAssetsPanel` or `SearchableCollectiblesPanel` are not ideal places to define the height limit. It's up to the component using those panels to manage sizing depending on the context.

In this case the max height is feature of `AssetSelector` and should be maintained there. It can be done e.g. in a following way:

```qml
    StatusDropdown {
        id: dropdown

        y: root.height + 4
        x: root.width - width

        width: 448

        closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutsideParent
        padding: 0

        contentItem: Item {
            implicitHeight: Math.min(searchableAssetsPanel.implicitHeight, 200)

            SearchableAssetsPanel {
                id: searchableAssetsPanel

                anchors.fill: parent

                objectName: "searchableAssetsPanel"

                function setCurrentAndClose(name, icon) {
                    button.name = name
                    button.icon = icon
                    button.selected = true
                    dropdown.close()
                }

                onSelected: {
                    const entry = ModelUtils.getByKey(root.model, "tokensKey", key)
                    highlightedKey = key

                    setCurrentAndClose(entry.symbol, entry.iconSource)
                    root.selected(key)
                }
            }
        }

        onClosed: searchableAssetsPanel.clearSearch()
    }
```
Where `200` should be a value taken from the design, potentially placed in `d`.

Reply:
https://github.com/user-attachments/assets/7e4c56a7-9d14-4db5-9409-2417c323ac8b

fixed as per your suggestion @micieslak